Classes/Notifications/NotificationSectionController.swift

```diff
@@ -81,6 +81,14 @@ final class NotificationSectionController: ListSwiftSectionController<Notificati
 
         BadgeNotifications.clear(for: model)
 
+        if model.type == .securityVulnerability {
```

Comment:
This seems like a good case for a `guard` 😄

Reply:
Sure thing :))